python/README.md

```diff
@@ -6,9 +6,17 @@ This folder contains examples and code snippets for the openEO Python client.
 
 Environments: `Python` (plain Python code), `Jupyter` (e.g. Notebooks)
 
+
 | Title | Environment | Description |
 | ----- | ----------- | ----------- |
-| ...   | `Python`    | ...         |
+| [Anomaly_Detection](./RescaleChunks/)   | `Python`  | Check the crop growth on your field and compare it with similar fields in the region.|
```

Comment:
These are all Python Notebooks as far as I can see, so the Environment should be set to `Jupyter` instead of `Python`